procat/settings/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Django settings for proCAT project."""
+
+from .settings import *  # noqa: F403
```

Comment:
Just to check your understanding. Do you know what this `noqa` comment is doing here?

Reply:
So that Ruff ignores the wildcard import?

Reply:
Yes, exactly